_posts/en/newsletters/2018-11-13-newsletter.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Bitcoin Optech Newsletter #21"
+permalink: /en/newsletters/2018/11/13/
+name: 2018-11-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a few discussions on the Lightning-Dev
+mailing list and suggests an opportunity to develop a new tool some
+users would find helpful.  Several notable code changes in popular
+Bitcoin infrastructure projects are also described.
+
+## Action items
+
+None this week.
+
+## News
+
+- **LN developer summit and mailing list activity:** before, during, and
+  after a planned meeting among Lightning Network protocol developers,
+  the [Lightning-Dev mailing list][] saw a surge of new proposals and
+  discussion about earlier proposals.  Below are some highlights:
+
+    - **Advertising node liquidity:** Lisa Neigut [proposes][neigut
+      liquidity] allowing LN nodes advertise that they're willing to
+      provide incoming capacity in exchange for a certain level of fees.
+      Merchants need their payment channels to have incoming capacity in
+      order to be able receive secure offchain payments from
+      customers---the current alternatives are either requiring some of
+      their customers to wait for several onchain confirmations to open
+      a new channel or making manual channel liquidity arrangements with
+      other merchants.  Although solving this problem would be highly
+      advantageous for merchant adoption of LN, it does pose some
+      technical challenges that discussion participants attempt to solve
+      both in this thread and in a [related thread][zmn liquidity].
+
+    - **Walletless channel opens:** pseudonymous contributor ZmnSCPxj
+      [notes][walletless opens] that if [BIP118][]
+      SIGHASH_NOINPUT_UNSAFE were adopted to allow implementing the
+      [Eltoo protocol][] in LN, then an LN implementation could set up a
```

Comment:
I don't think the wording "implementing the Eltoo protocol in LN" is quite right. Eltoo is presented as a entirely new protocol, not something to implement *in* LN (see the wording in the eltoo paper: "This is in stark contrast with the channel commitment invalidation procedure used today in Lightning", "This is in stark contract to the Lightning Network, where the reaction to a previous state being published needs to tailored to that specific state.", "In Lightning the information stored is asymmetric [...] With eltoo the information stored by the participants is symmetric")

This is a bit confusing, since ZmnSCPxj talks about "Lightning implementations" using SIGHASH_NOINPUT. Perhaps eventually we're just going to call all Payment Channel Networks "Lightning Networks", but for now I think it's clearer to maintain the distinction. I like "Payment Channel Network" or PCN as used in the Multi-hop lock paper (https://eprint.iacr.org/2018/472.pdf) since it's agnostic to the construction of the channels and routing, although perhaps PCN is not widely recognised.

Perhaps: "if [BIP118][]SIGHASH_NOINPUT_UNSAFE were adopted, as originally proposed in the [Eltoo paper][], then a Payment Channel Network implementation could..."

Reply:
I removed this summary.  As I was proofreading my description of Zmn's idea the other day, I [identified a possible flaw](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001556.html) in it, which Zmn [has now confirmed](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-November/001561.html).  This makes the idea less interesting, at least to me, and I think we have enough other news now that we don't need this item or a debate about where Eltoo fits in the PCN stack.  :-)

Reply:
> a debate about where Eltoo fits in the PCN stack

I was enjoying our debate! (but I'm sure we'll have an opportunity to revisit it at some point!)